build/build.js

```diff
@@ -9,8 +9,16 @@ const version = process.env.VERSION || require('../package.json').version
 const chokidar = require('chokidar')
 const path = require('path')
 
-const build = function (opts) {
-  rollup
+/**
+ * @param {{
+ *   input: string,
+ *   output?: string,
+ *   globalName?: string,
+ *   plugins?: Array<import('rollup').Plugin>
+ * }} opts
+ */
```

Comment:
I'm in VS Code, so the JSDoc comments help the TypeScript language server give me better intellisense. Plus this is self-documenting even if you don't have VSCode/TypeScript.

Reply:
Always great to have jsdoc comments. It helps contributing very smooth. 

Thanks 🎉

Reply:
Plus where you use the function, the editor can show you documentation, like in VS Code:

![Screenshot from 2020-06-07 10-58-03](https://user-images.githubusercontent.com/297678/83976281-ebf91680-a8ad-11ea-9b59-97726572f67f.png)